test_scripts/API/KeyboardEnhancements/Capabilities/003_HMI_doesnt_provide_KeyboardCapabilities.lua

```diff
@@ -0,0 +1,41 @@
+----------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0238-Keyboard-Enhancements.md
+----------------------------------------------------------------------------------------------------
+-- Description: Check SDL is still able to receive 'WindowCapabilities' from HMI without 'KeyboardCapabilities'
+-- and transfer them to App
+--
+-- Steps:
+-- 1. App is registered
+-- 2. HMI provides 'WindowCapabilities' without 'KeyboardCapabilities' within 'OnSystemCapabilityUpdated' notification
+-- 3. App requests 'DISPLAYS' system capabilities through 'GetSystemCapability'
+-- SDL does:
+--  - Provide 'WindowCapabilities' without 'KeyboardCapabilities' to App
+----------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/KeyboardEnhancements/common')
+
+--[[ Local Variables ]]
+local dispCaps = common.getDispCaps()
+dispCaps.systemCapability.displayCapabilities[1].windowCapabilities[1].keyboardCapabilities = nil
+
+--[[ Local Functions ]]
+local function check(_, data)
```

Comment:
```suggestion
local function checkKeyboardCapabilities(_, data)
```
I propose to rename the function

Reply:
I would agree if there are many "check" functions. However we have just only one in some scripts.